upload/catalog/model/catalog/product.php

```diff
@@ -536,4 +536,13 @@ public function getTotalProductSpecials() {
 			return 0;
 		}
 	}
+
+	public function checkProductCategory($product_id, $category_ids) {
+
+		$category_id = implode("', '", $category_ids);
+
+		$query = $this->db->query("SELECT * FROM " . DB_PREFIX . "product_to_category WHERE product_id = '" . (int)$product_id . "' AND category_id IN('$category_id')");
+
+		return $query->row;
```

Comment:
```return $query->rows;```

As for the IN statement, I would suggest the use of catalog/model/account/download.php file the way the $implode with AND, OR statement are being used originally.

Reply:
I am checking for the same.

Reply:
Not sure what that means but it seem you have a few 'for the same' in your replies there. Does this mean you'll make the same or you believe it's the same?